src/database/entities/member.rs

```diff
@@ -69,20 +70,21 @@ impl GuildMember {
             .await
             .map_err(Error::from)?;
 
-        let index = res.last_insert_id();
-        member.index = index as i32;
+        todo!("last_insert_id does not exist for Postgres");
```

Comment:
https://stackoverflow.com/questions/33382981/go-how-to-get-last-insert-id-on-postgresql-with-namedexec/37771986#37771986

Looks like we'll need to have the create query have 'RETURNING id' appended